controllers/profiles/unfollowUserV3.js

```diff
@@ -81,7 +131,7 @@ module.exports = async (req, res) => {
       activity_time: moment.utc().format('YYYY-MM-DD HH:mm:ss')
     });
   } catch (e) {
-    console.log('Error in unfollow user v2 scoring');
+    console.log('Error in unfollow user v3 scoring');
     return ErrorResponse.e409(res, e.message);
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to my previous comment, it would be beneficial to include the error message in the log output for better debugging.
```diff
- console.log('Error in unfollow user v3 scoring');
+ console.log('Error in unfollow user v3 scoring', e.message);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->